directory/datadog_checks/directory/directory.py

```diff
@@ -59,10 +60,11 @@ def check(self, instance):
 
             raise Exception("DirectoryCheck: the directory (%s) does not exist" % abs_directory)
 
-        self._get_stats(abs_directory, name, dirtagname, filetagname, filegauges, pattern, recursive, countonly)
+        self._get_stats(abs_directory, name, dirtagname, filetagname, filegauges, pattern, recursive, countonly, custom_tags)
 
-    def _get_stats(self, directory, name, dirtagname, filetagname, filegauges, pattern, recursive, countonly):
+    def _get_stats(self, directory, name, dirtagname, filetagname, filegauges, pattern, recursive, countonly, tags):
         dirtags = [dirtagname + ":%s" % name]
+        dirtags.extend(tags)
```

Comment:
no need to add an extra line, you can just do:
```python
dirtags = [dirtagname + ":%s" % name] + tags
```

Reply:
To get even more pythonic/pedantic:, whenever we switch to py3:
`dirtags = [dirtagname + ":%s" % name, *tags]`